### PR TITLE
Enable Helm chart release

### PR DESCRIPTION
**Pull request by mprimeaux**
### Description
-  Added change logs for the opensearch and opensearch-dashboards Helm charts.
- Amended README files to reflect the intended installation and usage.
- Incremented the version numbers to 1.0.2 for both Helm charts in adherence to linting rules and Semver 2.

@peterzhuamazon @TheAlgo Before merging this PR, please create a branch named `gh-pages` as per [these](https://github.com/opensearch-project/helm-charts/issues/55#issuecomment-925421426) steps. 

Also, I took the liberty of creating a README.md for the `gh-pages` branch, which you can see [here](https://github.com/mprimeaux/helm-charts/tree/gh-pages#readme).  You can see what it looks like by navigating to https://mprimeaux.github.io/helm-charts/

Once the `gh-pages` branch is created, I'm happy to submit a PR for the README if you prefer.

### Issues Resolved
This PR resolves issue #55. 
 
### Check List
- [:heavy_check_mark:] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by mprimeaux** [comment]
After following the steps as per above in my **forked repository**, I just wanted to confirm all works as expected:

```shell
❯ helm repo add opensearch https://mprimeaux.github.io/helm-charts
"opensearch" has been added to your repositories

❯ helm search repo opensearch
NAME                              	CHART VERSION	APP VERSION	DESCRIPTION
opensearch/opensearch             	1.0.2        	1.0.0      	A Helm chart for Kubernetes
opensearch/opensearch-dashboards  	1.0.2        	1.0.0      	A Helm chart for OpenSearch Dashboards
```

I don't mind changing the description of the OpenSearch Helm chart but I'd then need to increment the version to 1.0.3 or linting will fail.

**Comment by mprimeaux** [comment]
I modified the chart description for OpenSearch.

**Review by DandyDeveloper — APPROVED** [review]

**Comment by mprimeaux** [comment]
Please let me know if there is any other aspect to address. Happy to help. As a friendly reminder, the `gh-pages` branch would need to be created and configured before merging to `main` as per [these](https://github.com/opensearch-project/helm-charts/issues/55#issuecomment-925421426) steps.

**Comment by peterzhuamazon** [comment]
@TheAlgo please do another approve if you agree with the changes, I will create gh-pages branch once you approve.

Thanks.

**Comment by mprimeaux** [comment]
@peterzhuamazon @TheAlgo Thank you and appreciate your support.

File: README.md
Comment (TheAlgo) [inline README.md:29]: IMO , lets change the README in a separate PR after verifying the `helm search` command. I know it will work but still to prevent any dangling pointers it will be good.

File: charts/opensearch/README.md
Comment (TheAlgo) [inline charts/opensearch/README.md:18]: Same goes here , lets change this later.

**Review by TheAlgo — CHANGES_REQUESTED** [review]

File: charts/opensearch/README.md
Comment (mprimeaux) [reply charts/opensearch/README.md]: @TheAlgo I reverted the instructions to their previous value.

File: README.md
Comment (mprimeaux) [reply README.md:29]: @TheAlgo I reverted the instructions to their previous value.

**Review by TheAlgo — APPROVED** [review]
LGTM! @peterzhuamazon you can go ahead with merging after creation of the new branch. 

@mprimeaux we can change the README once we are good

**Comment by peterzhuamazon** [comment]
Thanks @mprimeaux @DandyDeveloper @TheAlgo I have created the gh-pages branch and merge this PR.

**Comment by mprimeaux** [comment]
@peterzhuamazon It appears there is a hook on the `gh-pages` branch that is causing a [failure](https://github.com/opensearch-project/helm-charts/runs/3737554862?check_suite_focus=true#step:5:56) in the releaser. Do you mind checking if there are any branch protection rules on the `gh-pages` branch and remove them?

**Pull request merged**
